mobile/src/main/java/org/hawkular/client/android/HawkularApplication.java

```diff
@@ -22,16 +22,21 @@
 import android.app.Application;
 import android.os.StrictMode;
 
+import com.squareup.leakcanary.LeakCanary;
+
 import timber.log.Timber;
 
 public final class HawkularApplication extends Application {
     @Override
     public void onCreate() {
         super.onCreate();
 
+        if (LeakCanary.isInAnalyzerProcess(this)) {
```

Comment:
@m-murad in case `LeakCanary.isInAnalyzerProcess(this)` turn true, `setUpLogging()` `setUpDetections()` and `setUpPush()` will not get executed. Is that the desired behaviour?